configuration/exodus-pipeline.yaml

```diff
@@ -130,6 +143,25 @@ Resources:
           - IsDev
           - Name: Build
             Actions:
+              - Name: Verify
+                ActionTypeId:
+                  Category: Build
+                  Owner: AWS
+                  Provider: CodeBuild
+                  Version: 1
+                Region: !Ref region
+                Configuration:
+                  EnvironmentVariables:
+                    !Sub '[
+                      {"name": "REPO_OWNER", "value": ${repoOwner}, "type": "PLAINTEXT"},
```

Comment:
I notice that the quoting isn't consistent between this place and a !Sub later on in the change.  e.g. you write ${repoOwner} here but write "${env}" in the other one.

I don't have a good understanding of how exactly the string in !Sub is evaluated, but if you tested this and it works then I guess neither one of them needs the quoting... which would make sense if this is being evaluated as YAML.

Reply:
This was a mistake; the value should be wrapped in double-quotes.

The sub function requires the ${xxxx} syntax but EnvironmentVariables requires a JSON format string.

You'll also see #{xxxx} a few times, this syntax is used for pipeline variables.
https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-variables.html